docs-ref-conceptual/azure-cli-configuration.md

```diff
@@ -60,62 +60,62 @@ Configuration files are written in the INI file format. This file format is defi
 
 Booleans are case-insensitive, and are represented by the following values.
 
-* __True__: 1, yes, true, on
-* __False__: 0, no, false, off
+* __True__: `1`, `yes`, `true`, `on`
+* __False__: `0`, `no`, `false`, `off`
 
 Here's an example of a CLI configuration file that disables any confirmation prompts and sets up logging to the `/var/log/azure` directory.
 
-`ini
+```ini
 [core]
 disable_confirm_prompt=Yes
 
 [logging]
 enable_log_file=yes
 log_dir=/var/log/azure
-`
```

Comment:
This will cause incorrect rendering:

![image](https://user-images.githubusercontent.com/4003950/103724940-76eeea80-5010-11eb-8e12-e281da81ced1.png)

Reply:
Splitting this issue to a separate PR https://github.com/MicrosoftDocs/azure-docs-cli/pull/2377 while this one is under debate.